src/services/expenses.service.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+let expenses = [];
+
+const getAll = () => {
+  return expenses;
+};
+
+const add = expense => {
+  expenses.push(expense);
+
+  return expense;
+};
+
+const getById = (expenseId) => {
+  const expense = expenses.find(({ id }) => id === +expenseId);
```

Comment:
```suggestion
  const expense = expenses.find(({ id }) => id === Number(expenseId));
```

The same in all other places, it's easier for reading